articles/overview/what-is-quantum-computing.md

```diff
@@ -0,0 +1,54 @@
+
+The theory of quantum physics posits that matter, at a quantum level is simultaneously in multiple states. And those many states interfere with each other like waves in a tide pool.
+
+Quantum computing stores information in quantum states of matter and uses quantum operations to compute on that information, by harnessing and learning to program quantum interference.
+
+Quantum computing might sound daunting, but with the right resources you can start building quantum applications today.
+
+## The qubit
+
+In quantum computing, a quantum bit is a unit of quantum information, like a classical bit. Where classical bits hold a single binary value such as a 0 or 1, a qubit can hold both values at the same time. This is called **superposition**.
+
+The act of **measurement** changes a qubit. It goes from being in the infinite states of superposition to one of the binary states of 0 or 1. Again, measurement of a qubit is unlike a classical bit, which when 0 is measured as 0, and when 1 is measured as 1.
+
+Qubits can also be **entangled**: when we make a measurement of one entangled qubit, it affects the state of the other, even if the qubits are physically far apart.
+
+## What can a quantum computer do?
+
+When multiple qubits act coherently, they can process multiple options simultaneously. This allows them to process information in a fraction of the time it would take even the fastest nonquantum systems.
+
+Quantum algorithms can be designed to take advantage of qubit superposition and entanglement to speed up classical algorithms, or to provide entirely new ways of modeling physical systems.
+
+One of the most famous quantum algorithms is `Shor's algorithm` for factorization, which makes the classically intractable problem of factorization of a large number into two prime numbers fast enough to challenge traditional cryptography.
+
+On the more constructive side, algorithms for secure cryptographic key distribution are made possible by superposition, quantum entanglement, and the **no cloning** property of qubits: the inability for qubits to be copied without detection.
+
+`Grover's algorithm` provides a quadratic factor speed-up for database searching.
+
+Using quantum programs to model quantum systems themselves is another application of quantum computing. Photosynthesis, superconductors, and chemical molecules are examples of quantum systems that can be simulated using quantum programs.
+
+## Quantum hardware
+
+In classical computers, bits correspond to voltage levels in silicon circuits. Quantum computing hardware can be implemented by many different physical realizations of qubits: trapped ions, superconducting, neutral atoms, electron spin, light polarization, topological qubits. Quantum hardware is an emergent technology. It requires balancing fidelity of the system with scalability. The larger the scale (that is, number of qubits), the higher the error rate.
```

Comment:
The beginning of this paragraph suggest that all classic computers are done with transistors, which maybe can lead to confusion. You can construct classical computers with any system that behaves according classical mechanics, for example with marbles or vaccum tubes. 
Personally I like the explanation how we use classical systems to represent classical bits and implement classical gates to then do the same for quantum computing but using quantum system to represent qubits and quantum gates.

Reply:
I like the conciseness/focus of the paragraph, so I'd personally say don't expand beyond ~one sentence referring to the classical stuff. But to clear up potential confusion it might help to just say something like the 1's and 0's of bits correspond to the on/off's of voltage levels (thanks to transistors). Because people might scratch their head and think "but wait, voltage levels can be in many different states too" and then oh no we'd have to explain analog computing too.